### DISCUSSION TITLE
Introduce Yourself 👇

### DISCUSSION THREAD

**Discussion by emlweb** [openclimatefix/community#2] [New to OCF]
### Welcome to the OCF Community! Say hi in the thread below 👋

Welcome! If you haven’t been here before, you've arrived at the Open Climate Fix community on Discussions! We’re happy to have you with us.

Please make yourself at home and start exploring all the different areas of the Community.

If you’d like to, introduce yourself below. You can share a little about who you are, what language you like to code in or want to learn more about, why you’re interested in Open Climate Fix, and/or what you hope to get out of the Community.

**Comment by pranavpurankar** [comment]
**Hi OCF team**,

I'm really excited about **OCF's** vision and would love to contribute to this amazing initiative.  I'm currently restarting my software career, and OCF's focus on sustainable climate solutions is incredibly inspiring - it's definitely something we all need to be thinking about!

An added bonus is that your projects use **Python**, which is my favorite language to work with.  I'm particularly interested in the **Open-Source-Quartz-Solar-Forecast** and **India-forecast app** products.  In fact, I've already submitted a proposal (part of GSoC) for them.

_Here's hoping that selection is based purely on skills and vision rather than university affiliation.  I'd be thrilled to be a part of the OCF team._

Thanks!

**Comment by aryanbhosale** [comment]
### Hi Community!

Glad to see the discussion thread! It's a crucial part of the community I am looking forward to join!

I'm [Aryan Bhosale](https://www.linkedin.com/in/aryan-bhosale-648323223), a third year undergrad(BE (hons.) ECE) at BITS Pilani, and I'm the webdev head of the [developers' society bits goa or devsoc](https://devsoc.club), where I manage our servers on GCP, host API services, build cool projects, and manage over 200 people passionate about dev. I love to build, code, and share. Doing something for the better good is something that I always look forward to doing, even in tech. I came across Open Climate Fix when I was exploring different orgs on the Google Summer of Code page. The objectives and the project style was fairly different from the other orgs that have been there for years, which is what caught my eye and appealed to me. 

I started contributing back in February this year (still going strong :)) and have made quite a lot of merges since then in various repos(my favourite one being the terraform commits since it's something very new to me :P). I have also been added as a contributor for code and docs. I can say I'm fairly experienced in programming as a whole, irrespective of the language(though the first language that I learnt 6 years ago is Python). 

I've communicated with many of the engineers, researchers, and devs here as I made my commits over the past 2 months. Peter, Zak, and Jacob, they're all very helpful and kind. I would love to get involved more and learn from them.

I would love to hear a feedback(if that's possible) on my [Google Summer of Code proposal](https://acrobat.adobe.com/id/urn:aaid:sc:AP:1dfc77c3-3d38-42d9-b682-62fba804f4a8) here for the Open Quartz Solar Forecast project.

Thank you again for this discussion thread!
Was really looking forward to something like this.

Cheers

**Comment by JasonFengGit** [comment]
### Hi! This is Zhenbang Feng.
I am a undergrad in University of Southern California studying Computer Science. I have been contributing to Satip and Open Quartz Solar Forecast. I have interned in TikTok's Recommendation Team, where I developed a personalized music recommendation system. My interests lie in software engineering and machine learning.

Please feel free to connect with me through my [LinkedIn](https://www.linkedin.com/in/zhenbang-feng) or check out my [GitHub](https://www.github.com/JasonFengGit).

**Comment by 14Richa** [comment]
Hi team, 

I am Richa Sharma, a software engineer based in London. Currently, I am pursuing a Master's in Computer Science from the Creative Computing Institute, University of the Arts London. I have been an active open-source developer, having previously participated in GSoC’23 with Postman and Outreachy Fellowship with Mozilla.

I like developing code in public and try to open source a lot of my personal projects as well.

I'm interested in working on the following issue - openclimatefix/Satip#222 during the GSOC program.

#### Why am I interested in this project :

I find the problem domain fascinating and impactful. Until now, I have only worked in dev tooling, this would be an interesting opportunity to work on scientific software development. It involved working with different APIs and edge cases while writing the wrappers for download managers. I feel this is very good training for software development. I am also interested in learning more about Climate modelling and I feel this would be a nice gateway for me to get my foot in the door where I can contribute as a software engineer and I can learn about climate research from other members of the team.

Additionally, the organization being based in London makes collaboration and contribution much more accessible for me. Moreover, as I'm currently considering thesis projects, I see an opportunity to align my academic pursuits with the work at Open Climate Fix. If selected, I could potentially extend my contributions to form part of my thesis research. I'm open to ideas and collaboration, and eager to work on the challenges within this domain that could advance both the OCF's objectives and my academic goals.

I have contributed a couple of PRs to the project already - 

1.  https://github.com/openclimatefix/Satip/pull/240
2. https://github.com/openclimatefix/Satip/pull/249

I will keep working on any other related PRs/issues as and when I find them. cc: @jacobbieker 

Feel free to reach out to me on [LinkedIn](https://www.linkedin.com/in/richa14/).

Cheers
Richa

**Comment by roshnaeem** [comment]
Hello everyone, excited to see this discussion channel :)) 

I am Rosheen Naeem. I am a Master’s student in Software Engineering for the Green Deal(SE4GD) at Vrije University Amsterdam, focusing on the impact of software on environmental sustainability. Currently, I am doing my thesis which focuses on applying machine learning (ML) and natural language processing (NLP) techniques to classify sensor data. I started contributing to open source in 2020 through hacktoberfest, and since then the journey has continued. I am excited to tell you that I have participated in [Outreachy](https://www.outreachy.org/) 2021 and [GSOD](https://developers.google.com/season-of-docs) 2021 which enhanced my passion towards open source.  

I was thrilled to find OCF in GSOC, which shares my aim to leverage software to combat climate change and reduce carbon emissions. I have been working in Python for a long time, and recently I have delved into the ML realm through various projects.  Working on these projects has made me more curious about what I can do with ML.

I submitted my GSOC proposal for "**Add other model architectures to Open Quartz Solar Forecast"**. I started contributing to OCF in February and still feel the same excitement every time I open a new PR 😁. 

Looking forward to working with OCF :))

**Comment by weiyang22** [comment]
Hello everyone, I'm Liue, currently pursuing a Master's degree in Computer Science and Technology at Lanzhou University. My primary research areas are machine learning and computer vision. In 2021, I participated in the Open Source Summer program organized by the Institute of Software, Chinese Academy of Sciences, which sparked my strong interest in open source. I proposed to "Add other model architectures to Open Quartz Solar" and have already submitted some preliminary test code with alternative datasets to the community in March. I'm looking forward to contributing to the community's work this summer.

**Comment by aavashsubedi** [comment]
Hi!

I am Aavash, currently finishing up my Master's in Physics at the University of Manchester and due to start a DPhil this September in AI for the Environment. I am primarly intersted in AI4Sciences and doing fundamental ML research inspired by problems in the natural sciences. Specfically looking at innovating/using Geometric Deep Learning (think graphs and geometry) & Foundational Models (Diffusion/Neural Operators) for weather predictions and material sciences!

I have worked on using Equivariance for Drug Discovery and applying GNNs for digital histopathology and currently in radio astronomy . I also particpated in ClimateHack last year so I am quite excited to get involved with OCF in any capactiy I can  : ) 

P.S. Always happy to nerd out Geometry, Graphs and Neural Operators so feel free to reach out!

**Comment by suleman1412** [comment]
Hello everyone, thrilled to see this discussion post! 

I'm Suleman, an open-source enthusiast from India. Discovering OCF through GSoC '24 sparked my interest and I dived right in with contributions. This is my first time contributing to open source, and I'm stoked to say a few of my contributions already got merged! It's been amazing, and I'm eager to keep learning and giving back. I'm excited to connect with the talented people at OCF and collaborate within this fantastic community!

**Comment by terapyo1304** [comment]
Hey guys!
I'm Aryaman, a student of artificial intelligence and machine learning at Manipal University Jaipur, India. I came across OCF in GSOC's organisations list. I've applied as a contributer for the **'Add Himawari/GOES imagery to Satip'** project and submitted the proposal for the same. I've done similar work with EU's **sentinel 2 satellite** during my time as an apprentice at dClimate Inc. I used **xarray, zarr and GitHub actions** for the same. [Sentinel pipeline ingestion](https://github.com/terapyo1304/Sentinel-pipeline-ingestion) You can find my work here!
I'm excited to work with y'all!
Cheers!

**Reply by terapyo1304** [reply]
@jacobbieker has been super supportive throughout the application process!
Cheers!

**Comment by MadJokkerr** [comment]
Hello @everyone
I'm Suvrat Patel, a bachelor's student with a deep passion for technology and machine learning. My expertise spans a range of programming languages including Python, C/C++, and Go, along with mastery in machine learning frameworks such as PyTorch, TensorFlow, and Keras and additional i have experience in Stacks like cybersecurity, cloud computing as well as in quantum Computing. I have applied these skills in various projects, from enhancing image quality using Generative AI to developing real time solution for hazard prevention Systems for Government satellite to developing time improvement pipelines from quantum registers.
In addition to my technical skills, my work experience as a Project Coordinator and Community Lead has endowed me with a unique blend of analytical, technical, and project management abilities. This combination has honed my ability to approach complex problems with innovative solutions, making me a versatile and effective contributor.
I am excited about the opportunity to leverage my background in machine learning projects, where I can contribute to impactful solutions and collaborate with like-minded individuals.
I look forward to the possibility of contributing my skills and enthusiasm to your team.
Best regards,
Suvrat Patel
LinkedIn : https://www.linkedin.com/in/suvrat-patel-45331aa7/

**Comment by glitch401** [comment]
Am I too late to the introduction thread?

If not, here goes my introduction:
Hi everyone, I'm Indranil, also known as Neil or glitch401. I'm currently pursuing my master's thesis in graph representational learning at the University of Texas at Dallas. I've accumulated three years of valuable experience in data science, having contributed to the teams at G2.com, Byjus, and Myntra. The mission statement of OCF has truly resonated with me. And my love for Nature, as I hike and trek a lot as my favorite pastime time.

Summer gets really hot, outdoors in Texas, and I'm limited from working in the industry. Hence I could not think of a better time to kick off my open-source contribution. I'm particularly eager to learn from and contribute to the OCF community. I discovered OCF through [GoodFirstIssues](https://goodfirstissues.com), [CodeTriage](https://www.codetriage.com) and GSoC 2024 project archives. 

I am looking forward to growing with the community!

**Comment by mduffin95** [comment]
Hey everyone!

I'm Matt, a software engineer based in Brighton, UK. I think I discovered OCF through a slightly different route. A friend of mine works for an organisation that's part of the Subak accelerator. When I was browsing through the other organisations, OCF piqued my interest. 

I've been involved with other organisations tackling climate change but thought that using my technical skills at OCF could be particularly effective!

P.s. I also attended your webinar last week which was quite informative, thanks.

**Comment by felix-se-cat** [comment]
Hi all!

Thanks for a very informative Office Hours session earlier today, I am Felix.
I am coming from a mechanical engineering background and have worked on an ML-based power prediction model for hydro turbines for my MSc. The whole idea of pulling as much knowledge as possible out of a dataset was really interesting to me, so now I am pursuing some more advanced ideas in the hydro sector in my own time.

I would be keen to contribute and become part of the community, will trawl through the different discussion boards and FAQs and then hopefully start contributing to some open issues.

Looking forward to it!

**Comment by broos27** [comment]
Hi Everyone! I'm Bradley, a first-year undergraduate student at Virginia Tech majoring in data analytics with a minor in meteorology and computer science. I'm very passionate in weather and climate, and have deepened this passion while gaining data analysis/visualization experience by volunteering at a local meteorological observatory for 2 years. Currently, I'm working on an open data science challenge of predicting urban heat index island temperatures with ML. I'm also quite interested in NWP, and have been following the impressive advances in this area over the past couple of years with AI-driven models such as FourCastNet and GraphCast.

In terms of my preferred languages, I am most well-versed in Python (and common data analysis libraries like Pandas, NumPy, etc.) and Java. I also have a bit of experience with MATLAB, R, and... Fortran 😅 (from my undergraduate research lab experience). 

Based on my experiences, contributing to OCF feels like a natural step for me to gain invaluable experience with open-source development. In particular, I'm interested in becoming a contributor for the GSoC 2025 program and look forward to connecting with the OCF community, exploring the project ideas, and submitting a PR where I can in the coming weeks!

**Comment by aayushyatiwari** [comment]
Hi everyone! I am Aayushya, and I am a second year student at Army Institute of Technology in Pune, India. I have been very fascinated with Data analysis and what deep learning could do. There's a lot more I have to learn about data. Which excites me. I really, REALLY appreciate what this organization does. It's something people overlook. And I am looking forward to contributing and learning.

**Comment by mrgear111** [comment]
Hi everyone,
I’m Daksh Saini, a B.Tech Computer Science student passionate about open-source and machine learning. I recently came across Open Climate Fix’s GSoC 2025 project ideas, and I’m excited about the opportunity to contribute!
I have experience with Python and some familiarity with PyTorch. I’d love to start contributing

**Comment by lambaaryan011** [comment]
Hi everyone! 🚀

I'm Aryan Lamba, a Computer Science student at Polaris School of Technology, passionate about web development, open-source contributions, and blockchain technology. I love building innovative digital solutions and optimizing performance.

My expertise includes Java, JavaScript, ReactJS, Node.js, SQL, and Python, and I enjoy creating scalable, user-focused applications. Lately, I’ve also been diving deep into Bitcoin protocol development and exploring blockchain systems.

As an active member of the tech community, I collaborate on impactful projects and constantly seek new challenges to grow as a developer. Let’s connect and build something amazing together!

🔗 LinkedIn: https://www.linkedin.com/in/aryan-lamba-1b2769292/
💻 GitHub: https://github.com/lambaaryan011

What’s the most exciting tech you’re currently exploring? Let’s chat in the comments!
@emlweb

**Comment by ajitashwath** [comment]
Hey everyone!

I'm Ajit Ashwath, a second-year B.Tech student specializing in AI & ML. I’ve been fascinated by ML technologies way back since before my 11th grade, and over the years, I’ve built a handful of projects exploring different aspects of it. I mostly code Python for ML-related project, TypeScript for some web based and most of my DSA, CP in C++ and I’m always looking for ways to push my skills further. That's how I got exposed to Open Climate Fix.

When I came across Open Climate Fix’s GSOC idea, "Quartz Solar: New Data Source," I got even more excited - it’s the perfect blend of AI and real-world impact. I see this opportunity as a way to work with you all and gaining the experience, skills, and global exposure needed to make that happen. 

Looking forward to learning as well as contributing.

**Comment by chaitra-samant** [comment]
Hi everyone! I’m Chaitra Samant, a second year student at VJTI Mumbai, passionate about Machine Learning and Open Source. I find Open Climate Fix’s mission inspiring, as it aligns with my interest in sustainable technology.

I have actively contributed to open-source projects, participated in Hacktoberfest, and even won open-source contests, which has strengthened my skills in collaborative development.

I’m particularly interested in contributing to:

- Open Data PVNet – Training an ML model for solar forecasting using open weather data.
- Adjuster this! TabPFN as a replacement for the adjuster – Exploring TabPFN to improve forecast adjustments dynamically.

I’m excited to work on PyTorch-based ML models and contribute meaningfully to these projects. Could someone guide me on how to get started? Are there specific tasks or beginner-friendly issues I can begin with?

Looking forward to collaborating and learning from the community! 🚀

**Comment by MadhavanAR** [comment]
Hello @everyone,

I'm Madhavan, a full-stack developer with expertise in Next.js, React.js, Node.js, and MongoDB, alongside deep knowledge in AI, LLMs, and agent-based systems. My focus lies in building scalable applications and integrating retrieval-augmented generation (RAG) with large language models (LLMs) to enhance AI-driven solutions.

Beyond web development, I specialize in prompt engineering and AI-powered automation. I enjoy experimenting with 3D web experiences using Three.js and optimizing backend workflows for high-performance applications.

Excited to connect and collaborate! 🚀

🔗 LinkedIn: https://linkedin.com/in/madhavan25

**Comment by ddwivedi2003** [comment]
👋 Hello Open Climate Fix Community!

I'm Dewashish Dwivedi, a pre-final year B.Tech student in Artificial Intelligence & Machine Learning from Pranveer Singh Institute of Technology, AKTU (graduating in 2026). I'm passionate about using AI for solving real-world problems in healthcare and climate.

This is my first time applying to Google Summer of Code, and I'm excited about the Open Data PVnet project. I’ve been exploring the repositories and learning about solar forecasting with NWP data. I’m still learning PyTorch but comfortable with Python and eager to contribute and grow with your guidance.

Looking forward to collaborating and learning from all of you! 🌞
GitHub: (https://github.com/ddwivedi2003) | (https://github.com/Dew2003)
LinkedIn: https://www.linkedin.com/in/dewashish-dwivedi-806a92206/

**Comment by ajey35** [comment]
👋 Hello Open Climate Fix Community!

I'm Ajeya Kumara K, a pre-final year B.E. Computer Science student at CMR Institute of Technology, Bengaluru (graduating in 2027). I’m passionate about using AI/ML to solve real-world challenges, especially in climate tech, sustainability, and energy forecasting.

This is my first time exploring GSoC, and I’m excited to learn more about OCF’s climate-focused AI projects. I’ve been exploring the docs and datasets, and I’m particularly interested in how machine learning and open data can enable better solar forecasting and support clean energy.

I’m comfortable with JavaScript, TypeScript, Python, Node.js, React, and SQL, and I’m actively strengthening my AI/ML skills. Looking forward to contributing, collaborating, and learning from this amazing community! 🌞🌱

GitHub: https://github.com/ajey35

**Comment by AmanKushwaha-17** [comment]
👋 Hello Open Climate Fix Community,

I’m Aman Kushwaha, an AI/ML developer from India.  
I’ve recently started exploring climate-focused machine learning, and OCF’s work in solar and weather forecasting caught my interest.

My background includes:
• Python and PyTorch  
• Time-series forecasting (mostly financial/market data)  
• Machine learning pipelines and backend APIs  
• Some experience with RAG/LLM-based applications  

I’m looking to slowly get familiar with OCF’s repositories and begin contributing in a meaningful way.  
If possible, I’d appreciate some guidance on:
• Which repository is recommended for new contributors  
• Any beginner-friendly issues I can look into  

GitHub: https://github.com/AmanKushwaha-17

Looking forward to learning from the community and contributing where I can.

**Comment by zer-art** [comment]
### Hi everyone! 👋

My name is **Pawan Paida**, and I’m joining from **India**. I’m currently a **3rd-year Data Analytics student** at **DSEU (Delhi Skill and Entrepreneurship University)**.

### 💻 Languages & Tech
I primarily code in **Python**, and my experience includes:
* **Libraries & Models:** TensorFlow, OpenCV, BERT models, LangChain.
* **Databases:** Vector Databases (Pinecone).
* **Tools & Workflows:** Conda environments, General Data Science & ML workflows.
* **Others:** Frontend tools (Streamlit, HTML/CSS), TradingView Pine Script.

Currently, I am looking to dive deeper into **MLOps**, **model deployment**, and **real-world climate-data applications**.

### 🌍 Why Open Climate Fix?
I joined this community because I want to use my data and machine-learning skills to contribute to impactful climate solutions. I’m especially fascinated by **forecasting problems** (like solar nowcasting) and really admire the open-source approach taken by OCF.

### 🚀 Goals
While I am here, I hope to:
* Make my first meaningful **open-source contribution**.
* Learn from experienced contributors and understand the workflow of large ML projects.
* Explore collaboration opportunities on OCF’s **Nowcasting** or related projects.

### 🔗 Connect with Me
* **LinkedIn:** [Pawan Paida](https://www.linkedin.com/in/pawan-parida-60134b276/)
* **Portfolio:** [My Projects](https://zer-art.github.io/portfolio/)

Looking forward to connecting with you all!

Best,
**Pawan Paida**

**Comment by revaarathore11** [comment]
Hi everyone ,
I’m Revaa Rathore, a first-year student at Medhavi Skills University. I’m really passionate about Machine Learning, Web Development, and Open Source, and I’ve already started exploring contributions in open-source projects.

I’m particularly interested in contributing to web tools, data pipelines, and documentation within OCF. With experience in Python, JavaScript, HTML, CSS, and React, I’m excited to learn, collaborate, and work on meaningful climate-focused projects.

Looking forward to contributing more and hopefully preparing myself for GSoC 2026 with OCF! 

Linkedin: https://www.linkedin.com/in/revaa-rathore-4aa555369/
github: https://github.com/revaarathore11

**Comment by Sharkyii** [comment]
Hello everyone! 👋 

I’m Sneh, a software developer and machine learning enthusiast passionate about environmental sustainability. I discovered Open Climate Fix through my interest in applying AI to climate science, and I was immediately drawn to OCF’s mission of improving renewable energy forecasting. Improving solar and weather forecasts is something I care about deeply, so I’m thrilled to be part of this community.

Some of my background and skills include:
- Machine Learning & Data: Proficient in Python, PyTorch, TensorFlow, Pandas, and scikit-learn for building and evaluating models. I’ve worked on time-series and geospatial data (such as satellite imagery) and love exploring new model architectures for prediction.
- Development & Tools: Comfortable with software development best practices—Git, Docker, CI/CD pipelines—and I’ve used cloud platforms like AWS/GCP for data workflows. I also have experience building APIs and data pipelines (for example, using xarray and netCDF for large scientific datasets).
- Climate & Energy Focus: I’m fascinated by solar energy and weather forecasting. For instance, I’ve looked into projects involving satellite data (GOES, Himawari) and open weather datasets for photovoltaic (PV) power forecasting. I’m eager to contribute to projects like Open Quartz Solar Forecast and Open Data PVNet to help improve solar forecasting models.

I’m excited to learn from everyone in the OCF community and to contribute where I can. Please feel free to connect with me on GitHub or LinkedIn—any guidance on getting started is much appreciated! Looking forward to collaborating and making an impact together.

Linkedin: https://www.linkedin.com/in/sneh-kansagara-b61362312/

**Comment by Srinath0916** [comment]
Hello everyone,

I’m Srinath Rao, a 2nd-year CS student, and I’m very interested in contributing to Openclimatefix. I’ve set up my environment, explored the repositories, and I’m eager to get involved.

If there are recommended starter issues, areas that need attention, or parts of the codebase that would benefit from additional documentation or testing, I’d love to help.

Excited to learn from the community and contribute wherever I can!
Thanks!

**Comment by SaitejaKommi** [comment]
Hi, I’m Saiteja, a second-year Computer Science student at Scaler School of Technology.
I’m interested in contributing to Open Climate Fix, particularly in areas related to software engineering and machine learning. I’m currently exploring the codebase and project ideas and look forward to contributing.

GitHub: https://github.com/SaitejaKommi

LinkedIn: https://www.linkedin.com/in/saiteja-kommi-19a962313/

**Comment by TenathDilusha** [comment]
Hi! I’m Dilusha, a 2nd-year CSE undergraduate at the University of Moratuwa, Sri Lanka. I’m interested in contributing to open-source projects and getting involved with the community, and I’m also aiming to participate in GSoC 2026. Excited to learn and contribute here!

**Comment by Prasun-13** [comment]
Hi everyone 👋,

I’m Prasun, a Computer Science & AI student from India.

I’m interested in contributing to Open Climate Fix, especially at the intersection of machine learning, data pipelines, and real-world impact.

My background is mainly in Python, and I’ve worked with:

- data preprocessing & validation
- ML workflows (training, evaluation, experimentation)
- reading and understanding large existing codebases

I’m particularly interested in learning more about:

- ML/data pipelines used in nowcasting and forecasting
- handling real-world scientific data (satellite, weather, PV, etc.)
- contributing through small, well-scoped issues first (docs, tests, validation, or simple fixes)

I’m exploring OCF with an interest in GSoC 2026, but my immediate goal is to contribute meaningfully, learn the codebase, and follow best open-source practices.

Looking forward to learning from the community and contributing where I can.

Thanks for building such impactful open-source projects!

— Prasun

**Comment by saun09** [comment]
**Hi everyone!** 

**I'm Saundarya Subramaniam**, a third-year Computer Engineering student at K.J. Somaiya College of Engineering. I’ve always believed that the intersection of AI and sustainability is the most impactful way forward in tech, which is what led me to OCF.

I have a strong background in Machine Learning and Full-Stack development, with a current CGPA of 9.5/10. Some of my relevant experiences include:

Machine Learning & Forecasting: I’ve worked extensively with TensorFlow and PyTorch. I recently developed "Fasal Forecast," a crop disease detection system using Inception V3. I also have experience with time-series forecasting using Prophet and Scikit-learn.
Data Engineering: In my recent internship, I built tools to clean and cluster 30,000+ rows of data using FastAPI and React, reducing preprocessing time by 50%.
Core Skills: I'm proficient in Python, Java, and C++, and I'm comfortable with GCP and Docker.


Looking forward, Cheers!

**Comment by workonlly** [comment]
Hey there! I'm Sujal, and I'm excited to contribute to OCF's GSOC 2026 journey.

I'm a Computer Science student and full-stack developer with experience in modern web technologies like Next.js, React, and Laravel, along with backend frameworks including FastAPI and Express.js. I've been actively building projects ranging from authentication systems to API integrations, and I'm passionate about open-source collaboration.

OCF's participation in GSOC 2024 and 2025 shows a strong commitment to the open-source community, and I'm eager to be part of this year's initiative. I'm particularly interested in contributing to projects that involve scalable architecture, API design, or developer tools—areas where I can leverage my experience while learning from the community.

Looking forward to connecting with the team and other contributors as more details emerge!

**Comment by Kavya751** [comment]
Hi everyone,

I'm Kavya, a Computer Science Undergraduate from India. 
I primarily work with Python and have experience in Data Analysis and Machine Learning. I came across Open Climate Fix while exploring organizations for GSOC 2026 and found the work around climate and renewable energy meaningful. I'm looking forward to understanding the projects better and contributing actively to the community!

**Comment by NidaNoushad** [comment]
Hi, I'm Nida Noushad. I am starting to explore OCF with the goal of contributing and preparing for GSOC 2026. For now I will be getting familiar with codebase and preparing to contribute

**Comment by sp-ankitprajapati** [comment]
Hi everyone!

I'm Ankit, currently doing my M.Tech in Climate Change at IIT Hyderabad.

I’ve been following OCF’s work on solar nowcasting because it overlaps heavily with my academic projects. I recently finished a project on modeling Indian Monsoon precipitation using ERA5 reanalysis data. I spent a lot of time wrestling with NetCDF files and Xarray to handle the high dimensionality (had to implement Ridge Regression from scratch to stop my RAM from exploding 😅).

I’m looking to contribute for GSoC '26 and would love to apply some of that time-series/satellite data experience here.

Since I'm already comfortable with the Python climate stack (xarray, pandas, cartopy), is there a specific repo (maybe pvnet or quartz?) where you need help with data pipelines or model benchmarking?

Here is my monsoon work if anyone is curious: [https://github.com/sp-ankitprajapati/Indian-Monsoon-Precipitation-Analysis]

Looking forward to learning from you all!

**Comment by jeevi** [comment]
Hello all,

My name is Chiranjeevi balawat and I'm a Senior engineer in SF bay area. I like to code in Python, Golang and Js. 

I've been in the industry for 10+ years and want to use my skills for the betterment of the future. 

Thank you.
